articles/search/search-security-overview.md

```diff
@@ -166,17 +166,18 @@ Currently, the only external resource that a search service writes customer data
 
 ### Exceptions to data residency commitments
 
-Although customer data isn't stored outside of your region, object names will appear in the telemetry logs used by Microsoft Support to troubleshoot your service issues. Object names are considered customer data. Names in telemetry logs include those of indexes, indexers, data sources, skillsets, containers, and key vault store.
+Although customer data isn't stored outside of your region, object names (considered as customer data), will appear in the telemetry logs used by Microsoft Support to troubleshoot your service issues. Telemetry logs includes names of indexes, indexers, data sources, skillsets, containers, and key vault store.
 
-Object names aren't obfuscated in the telemetry logs. If possible, avoid using names that convey sensitive information.
+>[!IMPORTANT]
+>Object names aren't obfuscated in the telemetry logs. If possible, please avoid using names that convey sensitive information.
 
-Telemetry logs are retained for one and a half years. During that period, support engineers might access and reference object names under these conditions:
+Telemetry logs are retained for one and a half years. During that period, support engineers might access and reference object names under the following conditions:
 
 + Diagnose an issue, improve a feature, or fix a bug. In this scenario, data access is internal only, with no third-party access.
 
 + Proactively suggest to the original customer a workaround or alternative. For example, "Based on your usage of the product, consider using `<feature name>` since it would perform better." In this scenario, Microsoft might expose an object name through dashboards visible to the customer.
 
-Upon request, Microsoft can shorten the retention interval or remove references to specific objects in the telemetry logs. Remember that if you request data removal, the trade off is reduced ability to troubleshoot any issues related to the object in question.
+Upon request, Microsoft can shorten the retention interval or remove references to specific objects in the telemetry logs. Remember that if you request data removal, Microsoft will reduce the ability to troubleshoot with historical information related to the object in question.
```

Comment:
How about "Microsoft will have reduced ability to"  -- otherwise, it sounds like Microsoft is intentionally reducing our ability to troubleshoot...  or if you don't like that phrase, you could say "Remember that if you request data removal, Microsoft won't have a full history of your service, which could impede troubleshooting."

Reply:
thank you, I like second suggestion 👍

Reply:
Angie, you have to make these edits in your branch and then push the changes. Line 180 should be:

"Upon request, Microsoft can shorten the retention interval or remove references to specific objects in the telemetry logs. Remember that if you request data removal, Microsoft won't have a full history of your service, which could impede troubleshooting."